pkgs/misc/emulators/openmsx/default.nix

```diff
@@ -1,48 +1,75 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, SDL2
+, SDL2_image
+, SDL2_ttf
+, alsa-lib
+, freetype
+, glew
+, libGL
+, libogg
+, libpng
+, libtheora
+, libvorbis
 , python
-, alsa-lib, glew, libGL, libpng
-, libogg, libtheora, libvorbis
-, SDL2, SDL2_image, SDL2_ttf
-, freetype, tcl, zlib
+, tcl
+, zlib
 }:
 
 stdenv.mkDerivation rec {
   pname = "openmsx";
-  version = "16.0";
+  version = "17.0";
 
   src = fetchFromGitHub {
     owner = "openMSX";
     repo = "openMSX";
     rev = "RELEASE_${builtins.replaceStrings ["."] ["_"] version}";
-    sha256 = "04sphn9ph378r0qv881riv90cgz58650jcqcwmi1mv6gbcb3img5";
+    sha256 = "sha256-9PdUNahJZ2O6ASkzLW/uudP3hiIzTDpxzFy6Pjb8JiU=";
     fetchSubmodules = true;
   };
 
-  nativeBuildInputs = [ pkg-config python ];
+  nativeBuildInputs = [
+    pkg-config
+    python
```

Comment:
can we use python3?

Reply:
Isn't it automatic? Anyway I will test this with py3 just to be sure.